common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -530,6 +536,11 @@ function(HTML5Video, HTML5HLSVideo, Resizer, HLS, _, Time) {
             this.el.trigger('pause', arguments);
         }
         this.el.trigger('ended', arguments);
+
+        if (document.getElementById("unit-next-button").disabled === true && document.getElementById("unit-next-button").previousState === true)
```

Comment:
what does the `previousState` check do?

Reply:
If the video component is the last component then next button is disabled by default and we shouldn't  enable it.

Reply:
No need of previousState now. Will remove it